src/frontend/src/components/NavBar/components/GroupDetailsDropdown/index.tsx

```diff
@@ -14,46 +19,43 @@ import {
 
 interface Props {
   anchorEl?: Element | null;
+  onClickInvite(): void;
   open: boolean;
 }
 
 const GroupDetailsDropdown = ({
   anchorEl,
+  onClickInvite,
   open,
 }: Props): JSX.Element | null => {
-  if (!open) return null;
+  const { data: userInfo } = useUserInfo();
+  const groupId = getGroupIdFromUser(userInfo);
+  const { data: members } = useGroupMembersInfo(groupId) ?? [];
+  const { data: groupInfo } = useGroupInfo(groupId);
 
-  // TODO (mlila): remove fake data
-  const groupInfo = {
-    id: 123,
-    name: "Santa Clara County",
-    location: "California/Santa Clara County",
-    memberCount: 4,
-  };
+  if (!open || !userInfo || !groupInfo) return null;
 
-  const groupInfo2 = {
-    id: 234,
-    name: "Santa Clara County and this group has a really really long name idk why",
-    location: "California/Santa Clara County",
-    memberCount: 6,
-  };
+  // how many people are in the current group
+  const memberCount = members?.length ?? 0;
 
-  const usersGroups = [groupInfo, groupInfo2];
-  const { memberCount, name, location } = groupInfo;
+  // right now users can only have one group, but will be able to have more in the future.
+  // ui already knows how to render for multiple groups, so we still want to give an array.
+  const usersGroups: Group[] = [groupInfo];
 
-  const userInfo = {
-    isOwner: true,
-  };
-  const { isOwner } = userInfo;
+  const { name, location } = groupInfo ?? {};
```

Comment:
will this unpack correctly if `{}`?

Reply:
yes, it will unpack to `undefined`, which is what I would expect.